plugins/services/src/js/components/forms/HealthChecksFormSection.js

```diff
@@ -127,27 +190,53 @@ class HealthChecksFormSection extends Component {
 
     const errors = errorsLens.at(key, {}).get(this.props.errors);
 
+    const endpointTooltipContent = (
+      <span>Select a service endpoint that you configured in Networking.</span>
+    );
+    const pathTooltipContent = (
+      <span>
+        Enter a path that is reachable in your service and where you expect
+        a response code between 200 and 399.
+      </span>
+    );
+
     return [(
       <FormRow key="path">
         <FormGroup
           className="column-6"
           showError={false}>
           <FieldLabel>Service Endpoint</FieldLabel>
-          <FieldSelect
-            name={`healthChecks.${key}.portIndex`}
-            value={String(healthCheck.portIndex)}>
-            <option value="">Select Endpoint</option>
-            {this.getEndpoints()}
-          </FieldSelect>
+          <Tooltip
```

Comment:
In other places, we have added these with a question mark icon next to the label text (inside the label). I would suggest that we do the same here.